### ISSUE TITLE
support for FOAF homepage

### ISSUE THREAD

**Issue by egonw** [enanomapper/data.enanomapper.net#63]
Please support linking out one the default GUI for homepage information, such as found, for example, for the nanomaterials in the SigmaAldrich and jRC bundles in the NanoWiki 3 data set.

**Comment by egonw** [comment]
@vedina where should I be looking in the source code, more or less, to get these links shown in the GUI in the info section (last column in the below screenshot)?

![image](https://cloud.githubusercontent.com/assets/26721/18080115/4b421806-6e94-11e6-8e0c-9d7a76d71c0e.png)

**Comment by vedina** [comment]
Links come from the external identifiers ; the UI is generally implemented as JavaScript [library](https://github.com/ideaconsult/Toxtree.js) outside of ambit code . @thejonan may help with enabling links.

**Comment by egonw** [comment]
The FOAF link is stored as external identifier under "HOMEPAGE". Two other links out I like to see made available on the website use ext IDs types "Same as" and "Close match".

@vedina @thejonan will it be possible to implement this half October latest? Because I like to know if it should and can be written up for D3.3.

**Comment by thejonan** [comment]
Yes, it is feasible. As long as the information is in the database, it'll be easy to just add it in the results. I'll need some briefing form @vedina on the overall context and will take care for this.

**Comment by egonw** [comment]
Some of the context is described in the attached
[integration-databases-data (7).pdf](https://github.com/enanomapper/data.enanomapper.net/files/451848/integration-databases-data.7.pdf)
 paper.

**Comment by egonw** [comment]
@vedina what if there is more than one foaf:page (or more than one owl:sameAs)? The current data model does not support that, I just found out... I can make a patch, but like a pointer on how to best do that... e.g. how do you solve this if there is more than one CAS number?

**Comment by vedina** [comment]
@egonw if you use external identifiers, it's not possible. CAS are stored as different object (properties of a structure)

**Comment by egonw** [comment]
What should I use? Does the system not break when I start hacking in some different object like for CAS?

**Comment by vedina** [comment]
Will think about. You can't just introduce new object, it will not be read/written to/from the database.

**Comment by egonw** [comment]
No, that's what I thought... so, are there other options for properties of substances with more than one value?
